src/app/grants/education/whoCanApply.tsx

```diff
@@ -0,0 +1,56 @@
+import ButtonWithArrow from "@/components/buttons/buttonWithArrow";
+import OutlineButton from "@/components/buttons/OutlineButton";
+import DynamicImage from "@/components/DynamicImage";
+import Paragraph from "@/components/Paragraph";
+import SectionHeader from "@/components/sectionHeader";
+import { useRouter } from "next/navigation";
+import BulletParagraph from "../bulletParagraph";
+
+export default function WhoCanApply({
+  setCurrentSection,
+}: {
+  setCurrentSection: (section: string) => void;
+}) {
+  const details = [
```

Comment:
We have to take out the language translation and research stuff below for now (we'll re-add once confirmed later) and only keep the dev eduction program gist